ably/proto/message_test.go

```diff
@@ -172,4 +230,40 @@ var _ = Describe("Message", func() {
 			})
 		})
 	})
+
+	acceptance := func(fixture string, t GinkgoTInterface) {
+		test, cfg := load(fixture, t)
+
+		It("fixture decode", func() {
+			for _, item := range test.Items {
+				err := item.Encoded.DecodeData(nil)
+				Expect(err).NotTo(HaveOccurred())
+				err = item.Encrypted.DecodeData(cfg)
+				Expect(err).NotTo(HaveOccurred())
+				Expect(item.Encrypted.Name).To(Equal(item.Encoded.Name))
+				Expect(item.Encrypted.Data).To(Equal(item.Encoded.Data))
+			}
+		})
+
+		return
```

Comment:
Yeah I saw it and thought that was some fancy new feature I didn't hear about ;).